src/main/java/org/jenkinsci/plugins/pagerduty/changeevents/ChangeEventBuilder.java

```diff
@@ -56,7 +68,19 @@ public boolean perform(AbstractBuild build, Launcher launcher, BuildListener lis
 		} catch (IOException | MacroEvaluationException | InterruptedException e) {
 			listener.getLogger().println("Error replacing summaryText tokens.");
 		}
-		new ChangeEventSender().send(integrationKey, expandedSummaryText, build, listener);
+		String expandedIntegrationKey = integrationKey;
+		try {
+			expandedIntegrationKey = TokenMacro.expandAll(build, listener, integrationKey);
```

Comment:
Added token macro expansion to this field so we can use secure credentials for the key vs directly entering into step.